modulemd/modulemd-module-stream.c

```diff
@@ -472,7 +494,8 @@ modulemd_module_stream_upgrade (ModulemdModuleStream *self,
   if (!mdversion)
     {
       /* If target mdversion is unspecified, set it to the latest */
-      mdversion = MD_MODULESTREAM_VERSION_LATEST;
+      /* TODO: reset to MD_MODULESTREAM_VERSION_LATEST */
+      mdversion = MD_MODULESTREAM_VERSION_TWO;
     }
```

Comment:
Should this TODO be updated?

Reply:
Yes, thanks for catching the temporary tweak that can now be reverted.